README.md

```diff
@@ -1 +1,63 @@
-# ironfoudation-team-project
+# WOOF application 
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Authors](#authors)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**Features include**
+- Secure store all of pet owners and pet sitters databases.
+- Regularly database update with personal information.
+
+**It also important to know:**
+To process the application some personal information about customers is required, for example, home address, mobile phone.
+
+## Technologies
+
+To install the application, please, see the list of requirements and restrictions settings below:
+ * Deployment target for iOS SDKs - 15.0
+ * Supported devices - iPhone only
+ * Supported modes - portrait
+ * Dark/night theme - deactivate
+ * The linters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.nters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.
+
+To contribute code, please, use:
+* macOS ventura 13.3
+* Xcode version: 14.3
+* Swift: 5.8
+* SDKs for iOS: 15.0
+* SwiftLint: 0.51.0
+* SwiftFormat: 0.51.7
+
+## Installation
+
+**Pre-requisites**: 
+To run this project: install Xcode (SDKs for iOS 15.0 as a deployment target).
+Then:
+
+1) clone the current repository via ssh: 
+
+```
+git@github.com:ios-course/ironfoudation-team-project.git
+```
+
+2) open the file Woof.xcodeproj in Xcode
```

Comment:
OK, I open the `Woof.xcodeproj` file and then what?
I still didn't install application. What should I do?